modules/core/src/main/java/org/apache/synapse/config/xml/SynapseXMLConfigurationSerializer.java

```diff
@@ -1,288 +1,288 @@
-/*

```

Comment:
This file has changed completely by this PR. Shall we revert that?

Reply:
Only an import was modified in this class:
```diff
-import org.apache.synapse.rest.API;
+import org.apache.synapse.api.API;
```
Even though the complete file appears as a diff here, this line is shown as the only diff in the IDE.